core/common/src/test/java/alluxio/util/ConfigurationUtilsTest.java

```diff
@@ -142,6 +143,15 @@ public void testMasterNotConfiguredMessage() {
         ConfigurationUtils.getJobMasterHostNotConfiguredMessage("test service 2"));
   }
 
+  @Test
+  public void parseAsList() {
+    assertEquals(Lists.newArrayList("a"), ConfigurationUtils.parseAsList("a", ","));
+    assertEquals(Lists.newArrayList("a", "b", "c"), ConfigurationUtils.parseAsList("a,b,c", ","));
+    assertEquals(Lists.newArrayList("a", "b", "c"),
+        ConfigurationUtils.parseAsList(" a , b , c ", ","));
+    assertEquals(Lists.newArrayList("a,b,c"), ConfigurationUtils.parseAsList("a,b,c", ";"));
+  }
```

Comment:
Can we add a case for `a,,b` to make sure empty strings are omitted?

Reply:
Good call. Added.